src/Modules/ContentBlocks/module.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module>
+  <name>ContentBlocks</name>
+  <version>5.0.0</version>
+  <requirements>
+    <minimum_version>6.0.0</minimum_version>
+    <maximum_version>7.0.0</maximum_version>
```

Comment:
Why this maximum version?

Reply:
Because we can't guarantee it will work in Fork7, while not strictly needed I think for core modules it probably also doesn't hurt